docs/source/getting-started/running-notebooks/index.rst

```diff
@@ -0,0 +1,139 @@
+most expedient if you're a Databricks customer, and another that is available to anyone, free of charge.
+
+Databricks Customers
+^^^^^^^^^^^^^^^^^^^^
+
+If you're already a customer of Databricks, you can use the MLflow service that is available as part of your Databricks workspace. 
+MLflow on Databricks is a fully managed service with additional functionality for enterprise customers, providing a scalable and secure managed deployment 
+of MLflow. Note that the MLflow APIs from within Databricks are the same as the open source version, so you can use the same code to run on Databricks or on 
+your own infrastructure.
+
+To learn how to get started using MLflow from within your Databricks account, please see the respective links below for your cloud provider:
+
+- `Azure Databricks <https://learn.microsoft.com/en-gb/azure/databricks/mlflow/>`_
+- `Databricks on AWS <https://docs.databricks.com/en/mlflow/index.html>`_
+- `Databricks on GCP <https://docs.gcp.databricks.com/en/mlflow/index.html>`_
+
+Databricks Community Edition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Databricks Community Edition (CE) is a fully managed, Databricks-hosted version of the Databricks platform. While many of the enterprise features of the 
+Databricks platform are unavailble on CE, the vast majority of MLflow functionality is supported (with the notable exception that you cannot create serving 
+endpoints on CE, so deployment of models is not supported).
+
+To get started with Databricks CE, please visit the `Databricks Community Edition page <https://docs.databricks.com/en/getting-started/community-edition.html>`_ 
+and follow the instructions outlined there. It takes about 5 minutes to get started, and you'll have a (mostly) fully functional Databricks Workspace that you 
+can use to log your tutorial experiments, runs, and artifacts to. 
+
+Once you log in to the Community Edition, you will see a landing page like this:
+
+.. figure:: ../../_static/images/tutorials/introductory/community-edition-main.png
+    :alt: Databricks CE Landing Page
+    :width: 100%
+    :align: center
+    
+    Databricks CE Landing Page
+
+In order to get to the MLflow UI, you can navigate to it by clicking on the "Experiments" link on the left-hand side (denoted by the laboratory beaker icon). 
+When you get to the MLflow UI on CE for the first time, you'll see this:
+
+.. figure:: ../../_static/images/tutorials/introductory/mlflow-ui-in-ce.png
+    :alt: Databricks CE MLflow UI
+    :width: 100%
+    :align: center
+    
+    Databricks CE MLflow UI
+
+Decisions about where to run your Notebook
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With a Databricks managed instance of MLflow, you have two options on how you'd like to proceed with using the managed Tracking Server instance. You can 
+import any notebooks that you'd like and use them from within CE directly, or you can execute the tutorials locally from your computer and simply login to CE 
+from within your notebook. 
```

Comment:
Could we make below those two options toggle lists? Then this section can be removed as it's explaining we have two options.

.. details::
   :open: false

   **Importing Notebooks directly into CE**

   Content for Toggle List Item 1.

   **Running locally and using CE as a remote tracking server**

   Content for Toggle List Item 2.

Reply:
sphinx doesn't have support for this, but I converted this to raw html to create the detail summaries with dropdowns.